src/components/map_rotator/index.js

```diff
@@ -35,8 +37,18 @@ var MapRotator = Base.extend({
     }
 
     var heading = - e.geoposition.coords.heading;
-    heading && this.map.el && this.map.el.animate({ rotate: heading + 'deg' }, 1000, 'ease-in-out');
+    var diff = heading-this.last_heading
```

Comment:
You dropped the `this.map.el` check. Is there a special reason why you did this?